controllers/articlesController.js

```diff
@@ -12,10 +12,17 @@ function getArticleById(request,response,next){
 }
 
 function getAllArticles(request,response,next){
-    selectAllArticles()
+    const {query} = request
```

Comment:
Is this used

Reply:
I used it to check query in the model - anything not sort_by or order is rejected 400